src/AccessibilityInsights.Win32/Properties/AssemblyInfo.cs

```diff
@@ -37,6 +37,7 @@
 [assembly: InternalsVisibleTo("AccessibilityInsights.SharedUx")]
 [assembly: InternalsVisibleTo("AccessibilityInsights.CommonUxComponents")]
 [assembly: InternalsVisibleTo("AccessibilityInsights.Extensions")]
+[assembly: InternalsVisibleTo("UITests")]
```

Comment:
My first thought was "Why aren't we doing this in the signed build?" I figured it out after I thought about it, but adding a comment here might clarify it for future readers

Reply:
Comment added